study.md

```diff
@@ -85,7 +91,7 @@ Using Hash methods and string interpolation in Ruby, create a string using
 conspiracy theories.".
 
 ```ruby
-# your answer here
+"My name is #{dale[:given_name]} #{dale[:family_name]} and I'm an #{dale[:occupation]} that enjoys conspiracy theories."
```

Comment:
`conspiracy theories` should have been accessed from `hobbies` if it had been set.